pom.xml

```diff
@@ -537,7 +553,14 @@
 				<plugin>
 					<groupId>org.apache.maven.plugins</groupId>
 					<artifactId>maven-enforcer-plugin</artifactId>
-					<version>1.4.1</version>
+					<version>3.0.0-M2</version>
```

Comment:
Anything we might need to consider before bumping these versions? Can they be transitive in some way that might lead to other issues?

Should be consider the latest version for 2.x instead of new major version?

Reply:
See comments above. Also, as you noted, this is the version that has been used in the `gs14` branch for `maven-enforcer-plugin`.